.scalafmt.conf

```diff
@@ -5,6 +5,8 @@ align {
   openParenCallSite = false
   openParenDefnSite = false
   tokens.add = [
+    "%",
+    "%%"
```

Comment:
I haven't used scalafmt before, but aren't you missing a comma _(`','`)_ at the end of this line?

Reply:
oh, yes